web/sites/default/settings.php

```diff
@@ -794,6 +794,16 @@
  'prefix' => '',
 ];
 
+/**
+ * Skip file system permissions hardening.
+ */
+$settings['skip_permissions_hardening'] = TRUE;
+
+/**
+ * Exclude modules from syncing
+ */
+$settings['config_exclude_modules'] = ['memcache'];
```

Comment:
What's going on with the `memcache` module? It's both removed and ignored …

Reply:
See discussion  above.